### DISCUSSION TITLE
Discord Channel

### DISCUSSION THREAD

**Discussion by samuelcastro** [UnlyEd/next-right-now#268] [Ideas] [answered]
Since NRN is growing up I think it'd be really helpful to have a [Discord](https://discord.com/) channel for it, where users and maintainers could have a more comprehensive dev experience to discuss issues, features, ideas, etc.

Thoughts?

**Comment by Vadorequest** [comment, marked as answer]
I think it's still too early, there isn't a big active community around NRN at this time. Also, NRN is just a Next boilerplate, so I think it'd make sense to use the Next.js discord instead.

**Reply by Vadorequest** [reply]
I think using Next.js Discord is the best option at the time, there is a strong and active community there, it'd also promote NRN to other people, increasing its reach. And people who ask questions that are actually related to Next would find their answers there too.

At some point, if the NRN community becomes too big it'll be the sign we need our own Discord.
